scripts/setup-centos8.sh

```diff
@@ -25,27 +37,32 @@ export CXXFLAGS=$CFLAGS  # Used by boost.
 export CPPFLAGS=$CFLAGS  # Used by LZO.
 CMAKE_BUILD_TYPE="${BUILD_TYPE:-Release}"
 BUILD_DUCKDB="${BUILD_DUCKDB:-true}"
+export CC=/opt/rh/gcc-toolset-9/root/bin/gcc
+export CXX=/opt/rh/gcc-toolset-9/root/bin/g++
```

Comment:
I think these should only be set when devtool-set is activated, so further down in the `if`

Reply:
These paths are present when the devtool-set is installed. The activation of devtool-set further exports other variables and paths.